docs/dcp2_system_design.rst

```diff
@@ -717,17 +759,43 @@ descriptors, one for metadata files and one for ``links.json`` files.
        The DSS adapter uses the DSS version of the metadata file,
        converting it to this restricted ISO syntax.
 
+   A staging area may only contain exactly one entity object with a given
```

Comment:
Yes. The same restriction applies independently to descriptors and I will add a sentence here.

Reply:
More importantly, I wonder if this breaks your code. When you rerun the analysis in a workspace, with a new value for the version command line argument, wouldn't you create a second object with the same uuid and a different version. If that is the case, we should lift this constraint for non-delta staging areas. I don't want to write spec that's not consistent with a current implementation.

@clairerye @aherbst-broad

Reply:
Yes, when we re-run with a new version argument we create a second object with the same entity_id and a new version. We only do this if another version has already been imported(otherwise we overwrite the original). In the past we have been careful to clean up the originals before creating new versions, but it would be easier if the constraint was relaxed for non-delta staging areas

Reply:
Modified spec to explicitly allow multiple versions of the same entity or subgraph in a non-delta staging area.